server/documents/modules/rating.html.eco

```diff
@@ -83,6 +83,29 @@ themes      : ['Default']
       <br><br>
     </div>
 
+    <div class="example">
+      <h4 class="ui header">Partial Ratings </h4>
+      <p>You can use decimal values for showing partially filled ratings</p>
+      <div class="ui ignored warning message">
+        <p>Partial ratings are only supported by browsers with support for <a href="https://caniuse.com/#feat=css-variables">CSS Variables</a> and <a href="https://caniuse.com/#feat=mdn-css_types_image_gradient_linear-gradient_interpolation_hints">Gradient Midpoints</a>. Unsupported browsers will use a rounded down value</p>
```

Comment:
You should also add an info that jquery >3.2 is necessary as well (for people which use FUI from a CDN, they do not come across dependency warnings)

Reply:
@zombietfk bump